source/iostreams.tex

```diff
@@ -2633,7 +2633,7 @@
 the function signature
 \tcode{basic_ostream\& stream::operator\shl(ios_base\& (*)(ios_base\&))}
 to permit expressions of the form
-\tcode{cout \shl dec}
+\tcode{cout \shl{} dec}
```

Comment:
The macros are defined as:

  \newcommand{\shl}{<{<}}
  \newcommand{\shr}{>{>}}

I tried removing the extra curlies, and it changes the order of some index entries. So I guess that's what they're for.